frontend/src/entry/settings/ManualEntryAccess.tsx

```diff
@@ -0,0 +1,67 @@
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore'
+import { Accordion, AccordionDetails, AccordionSummary, Button, Stack, TextField, Typography } from '@mui/material'
+import { getUserInformation } from 'actions/user'
+import { useState } from 'react'
+import MessageAlert from 'src/MessageAlert'
+import { CollaboratorEntry, EntityKind } from 'types/types'
+import { getErrorMessage } from 'utils/fetcher'
+
+interface ManualEntryAccessProps {
+  accessList: CollaboratorEntry[]
+  setAccessList: (accessList: CollaboratorEntry[]) => void
+}
+
+export default function ManualEntryAccess({ accessList, setAccessList }: ManualEntryAccessProps) {
+  const [manualEntityName, setManualEntityName] = useState('')
+  const [errorMessage, setErrorMessage] = useState('')
+
+  const handleAddEntityManuallyOnClick = async () => {
+    setErrorMessage('')
+    if (manualEntityName !== undefined && manualEntityName !== '') {
+      if (accessList.find((collaborator) => collaborator.entity === `${EntityKind.USER}:${manualEntityName}`)) {
+        return setErrorMessage(`The requested user has already been added below.`)
```

Comment:
The backend should handle this, not the frontend. An API user can bypass this check.

Reply:
This is UI validation to make sure you're not adding the same user to the form below, I think it's still worthwhile having.